packages/components/src/draggable/index.js

```diff
@@ -148,6 +149,19 @@ class Draggable extends Component {
 
 	render() {
 		const { children, className } = this.props;
+		if ( typeof children === 'function' ) {
+			return children( {
+				onDraggableStart: this.onDragStart,
+				onDraggableEnd: this.onDragEnd,
+			} );
+		}
+
+		deprecated( 'wp.components.Draggable as a DOM node drag handle', {
+			version: 3.10,
```

Comment:
`4.0` comes after `3.9` in WordPress world :)

Reply:
oups forgot about that :)

Reply:
Fixed in 53b10d3ffec11b4d6eacc57f98639eab86852ce4